media-radio/unixcw/unixcw-3.6.0-r2.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+
+DESCRIPTION="Collection of programs that fit together to form a morse code tutor program"
+HOMEPAGE="https://unixcw.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+IUSE="alsa ncurses pulseaudio suid test qt5"
+RESTRICT="!test? ( test )"
+
+RDEPEND="ncurses? ( sys-libs/ncurses:= )
+	qt5? ( dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5 )
+	alsa? ( media-libs/alsa-lib )
+	pulseaudio? ( media-libs/libpulse )"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig
+	sys-devel/libtool"
+
+src_prepare() {
+	append-cflags -std=gnu11
+	append-cxxflags -std=gnu++11
+	eapply -p0 "${FILESDIR}"/${PN}-3.6-tinfo.patch
+	# Bug# 837617 and 858278
+	sed -i -e "s/curses, initscr/ncurses, initscr/" \
+		-e "s/_curses_initscr/_ncurses_initscr/" configure.ac || die
+	eapply_user
+	eautoreconf
+}
+
+src_configure() {
+	econf --libdir=/usr/$(get_libdir) \
```

Comment:
Since you specify ```/usr``` manually, shouldn't be ```EPREFIX``` added?

Reply:
Correct, added. Thanks.